src/pharmpy/plugins/nlmixr/sanity_checks.py

```diff
@@ -62,9 +61,24 @@ def check_model(
         print_warning("Omega with value same not supported. Updated as follows.")
         model = change_rvs_same(model, omega=True)
 
+    # Checks regarding esimation method
+    if method := known_estimation_method(model):
```

Comment:
This is confusing. The function will return the name of an unknown estimation method and `None` if it is known. Perhaps this could be made clearer

Reply:
I agree, the function has been changed to be a bit clearer